ansible/templates/etc/php/7.2/fpm/pool.d/pool-template.conf.j2

```diff
@@ -0,0 +1,18 @@
+; {{ ansible_managed }}
+
+[{{ item.name }}]
+user = {{ item.user }}
+group = {{ item.user }}
+listen = /var/run/php/php-fpm-{{ item.name }}.sock
+listen.owner = www-data
+listen.group = www-data
```

Comment:
What is the difference between `user`/`owner` and `listen.owner`/`listen.group` and why do we need to set it this way?

Reply:
`user`/`group` apply to the user the PHP workers run under, and `listen.owner`/`listen.group` apply to the file permissions of the PHP socket. The latter needs to be accessible by nginx, and is therefore set to `www-data`.